content/language/word-list.md

```diff
@@ -44,7 +54,7 @@ These recommendations are broken into two groups. The preferred group does not u
 Preferred: 
 
 * Control plane/control plane node
-* Controller/doer
```

Comment:
For now, I'd like to remove any changes to the recommendations itself. I'm happy to accept changes to the language/justification however.